modules/linear_boltzmann_solvers/b_discrete_ordinates_solver/sweep/angle_set/aah_angle_set.cc

```diff
@@ -116,35 +114,35 @@ AAH_AngleSet::ReceiveDelayedData()
 }
 
 const double*
-AAH_AngleSet::PsiBndry(uint64_t bndry_map,
-                       unsigned int angle_num,
-                       uint64_t cell_local_id,
-                       unsigned int face_num,
-                       unsigned int fi,
-                       int g,
-                       size_t gs_ss_begin,
-                       bool surface_source_active)
+AAH_AngleSet::PsiBoundary(uint64_t boundary_map,
```

Comment:
Would `boundary_id` be valid here? `boundary_map` is a little unclear IMO.

Reply:
I agree. I think boundary_id is more appropriate.  Fixed.